### DISCUSSION TITLE
Recipes are not localized in setup screen

### DISCUSSION THREAD

**Discussion by pgy866** [OrchardCMS/OrchardCore#10396] [General]
![截图_选择区域_20211002232014](https://user-images.githubusercontent.com/28694676/135722680-746737a2-49f3-4723-9d43-3cb37e071dfb.png)

**Comment by hishamco** [comment]
While the recipes are sort of dynamic data, so they will not be localized yet. Please check this #4466 it's WIP, but if you want to localize them at your end, you could chnage the names in the recipe files

**Reply by pgy866** [reply]
@hishamco Thank you, I have adopted the browser translation method, which is a bit obsessive-compulsive. .

**Reply by pgy866** [reply]
Features? Is this also dynamic data?
https://crowdin.com/translate/orchard-core/all/en-zhcn?filter=basic&value=15

And did not find its content

![截图_选择区域_20211003001419](https://user-images.githubusercontent.com/28694676/135724408-8f8bbb93-d1ac-4759-91ae-76da9cb388ff.png)
s?

**Reply by hishamco** [reply]
Nope, the Dynamic Localization is Wait In Progress (WIP) not included yet
